libs/blocks/preflight/panels/martech.js

```diff
@@ -0,0 +1,48 @@
+import { html, signal, useEffect } from '../../../deps/htm-preact.js';
+import { createTag } from '../../../utils/utils.js';
+
+const martechBlock = signal(null);
+
+function getTable(strings) {
+  const table = document.createElement('table');
+  table.setAttribute('border', 1);
+  const headerRow = document.createElement('tr');
+  headerRow.append(createTag('th', { colspan: 2, style: 'width: 100%' }, 'martech metadata'));
+  table.append(headerRow);
+  strings.forEach((str) => {
+    const tr = document.createElement('tr');
+    tr.append(createTag('td', { colspan: 1 }, createTag('h3', {}, str)));
+    tr.append(createTag('td', { colspan: 1 }, createTag('h3', { 'data-ccp-parastyle': 'DNT' }, str)));
```

Comment:
Nit: Any reason you're using h3's?

If no real reason I could see bumping the font size down (16px?) so the table isn't as long.

Reply:
Under the hood in Word, the DNT style produces an H3. So this is just to try and pre-align with it before dropping the block in there.